tests/apps/workflowsapp/app.go

```diff
@@ -242,6 +256,6 @@ func HelloTemporalWF(ctx workflow.Context) (string, error) {
 }
 
 func HelloTemporalAct(ctx context.Context) (result string, err error) {
-	time.Sleep(8 * time.Second) // This 5s sleep is to allow for terminate to be called
+	time.Sleep(30 * time.Second) // This 5s sleep is to allow for terminate to be called
```

Comment:
Comment says 5s; it was 8s and now it's 30s.

Is there a way for us to get away without sleeping here?

1. It makes tests slower
2. It introduces flakiness: pretty much every test we have that depends on waiting for a condition to happen is flaky at some point (and I presume that's the reason why it went from 5, to 8, to 30s)

Reply:
I'll update the comment. The time is arbitrary since a terminate call is executed which will stop the activity all together. I'm using 30s here (I'll downgrade to 10s) in order to ensure that there is enough time for the activity to start and all the calls to be made. I do agree that 30s is excessive.